src/routes/overviews/bufnmf.svx

```diff
@@ -0,0 +1,134 @@
+---
+title: Audio Decomposition using BufNMF
+blurb: |
+    Decompose an audio buffer into component parts using non-negative matrix factorization.
```

Comment:
Another british english nit pick

Reply:
Factorization is with an "s"? FWIW British spellings don't trigger red flags in my mind so I can't say that I'll really be catching them. I'll sure try!!

Reply:
No probs :P I'll flag them where I see them and eventually write a test for it :P

Reply:
A test for British english! That's some machine assistance right there.